sktime/forecasting/tests/test_all_forecasters.py

```diff
@@ -123,6 +123,12 @@ def test_y_invalid_type_raises_error(Forecaster, y):
 def test_X_invalid_type_raises_error(Forecaster, X):
     """Test that invalid X input types raise error."""
     f = _construct_instance(Forecaster)
+
+    if f.get_tag("scitype:y") == "univariate" or f.get_tag("scitype:y") == "both":
+        y_train = _make_series(n_columns=1)
+
+    elif f.get_tag("scitype:y") == "multivariate" or f.get_tag("scitype:y") == "both":
```

Comment:
when we reach this line, it can't be that `f.get_tag("scitype:y") == "both"` because that already is checked in the first line.
So it's not wrong to have it in the condition, but redundant, since the second block will only be entered if `f.get_tag("scitype:y") == "multivariate"`. I would simplify this.

Reply:
Thank you Franz. I will edit it